src/coreclr/jit/compiler.h

```diff
@@ -8224,6 +8192,29 @@ class Compiler
         return eeGetEEInfo()->targetAbi == abi;
     }
 
+#if defined(FEATURE_EH_X86_FRAMES)
+    bool UsesFunclets()
+    {
+        return IsTargetAbi(CORINFO_NATIVEAOT_ABI);
+    }
+
+    bool UsesCallfinallyThunks()
+    {
+        // Generate call-to-finally code in "thunks" in the enclosing EH region, protected by "cloned finally" clauses.
+        return UsesFunclets();
+    }
+#else
+    bool UsesFunclets()
```

Comment:
nit
```suggestion
    static bool UsesFunclets()
```

Reply:
This is semi-intentional. The compiler will inline it for non-x86 configurations anyway, so the `static` doesn't have an effect on the resulting code. The absence of `static` does, however, prevent it from calling `Compiler:: UsesFunclets()` by accident and only discovering it when the x86 JIT is built on CI.